libcontainer/seccomp/patchbpf/enosys_linux_test.go

```diff
@@ -299,7 +299,7 @@ func TestEnosysStub_SingleArch(t *testing.T) {
 }
 
 func TestEnosysStub_MultiArch(t *testing.T) {
-	for end := 0; end < len(testArches); end++ {
+	for end := range len(testArches) {
 		for start := 0; start < end; start++ {
```

Comment:
Can't this also be

```suggestion
		for start := range end {
```

?

Reply:
Thanks for your sharpness, @cyphar; I fixed that.